telegram/ext/_application.py

```diff
@@ -78,6 +79,19 @@
 _STOP_SIGNAL = object()
 _DEFAULT_0 = DefaultValue(0)
 
+
+# Since python 3.12, the coroutine passed to create_task should not be an async generator. Remove
+# this check when we drop support for python 3.11.
+if sys.version_info >= (3, 12):
+    _CoroType = Union["asyncio.Future[object]", Awaitable[RT]]
+    _ErrorCoroType = Optional[Union[Optional["asyncio.Future[object]"], Awaitable[RT]]]
```

Comment:
Can anyone confirm these are the correct type hints?

Reply:
Will try to have a oder look on sunday